### ISSUE TITLE
prettier-java 1.5.0 deletes code

### ISSUE THREAD

**Issue by eladeyal-intel** [jhipster/prettier-java#514]
**prettier 2.2.1** + **Prettier-Java 1.5.0**

```sh
# Options (if any):
--print-width 80
```

**Input:**

```java
package com.demo;

public class C {

  private void f() {
    someList
      .stream()
      .filter(s -> serverUUID.equals("1234"))
      .forEach(
        s -> {
          s.set(status.toString());
        }
      );
  }
```

**Output:**

```java
package com.demo;

public class C {

  private void f() {
    someList
      .stream()
      .filter(s -> serverUUID.equals("1234"))
      .forEach(s -> );
  }
}
```

**Expected behavior:**

not delete code? whatever was after `s ->` was deleted

**Comment by RepubIique** [comment]
Can confirm, was fixed in prettier 2.4.1

**Comment by clementdessoude** [comment]
Thank you for reporting the issue ! I am closing this as the issue seems to be resolved :)
